subprojects/wrapper/wrapper.gradle

```diff
@@ -26,18 +26,9 @@ dependencies {
     integTestRuntime rootProject.configurations.testRuntime.allDependencies
 }
 
-ext {
-    wrapperBuildReceipt = "build-receipt.properties"
-}
-
-task wrapperBuildReceiptResource() {
-    inputs.property 'version', { version }
-    outputs.file { "${generatedTestResourcesDir}/${wrapperBuildReceipt}" }
-    outputs.cacheIf { true }
-    doLast {
-        generatedTestResourcesDir.mkdirs()
-        new File(generatedTestResourcesDir, wrapperBuildReceipt).text = "versionNumber=${version}"
-    }
+task wrapperBuildReceiptResource(type: Copy) {
+    into generatedTestResourcesDir
+    from project(':versionInfo').createTestBuildReceipt
```

Comment:
are the two changes that we put the wrapper build-receipt into a different location _and_ it contains different information? It looks like the `wrapperBuildReceiptResource` only contained the version number.